devel.config

```diff
@@ -1,16 +1,21 @@
-backend_port = 7001
-frontend_port = 7002
+command: "run"
```

Comment:
you can choose not to set the command in the config file, or you can override a default command on the command line, like this:

    thentos --command=runA3

proper sub-commands (as git knows them up to 3 levels or so deep) will be implemented at some point.

(it may also be a reasonable requirement in some settings to "ban" specific sub-configs from specific sources as you suggest.  i will put that on my list, but unless i have a good use case it will probably not make it to the top.)